diff -up gnome-menus-3.8.1/layout/gnome-applications.menu.fixed gnome-menus-3.8.1/layout/gnome-applications.menu
--- gnome-menus-3.8.1/layout/gnome-applications.menu.fixed	2013-12-03 16:38:08.896166629 -0500
+++ gnome-menus-3.8.1/layout/gnome-applications.menu	2013-12-03 17:06:31.143842549 -0500
@@ -42,7 +42,6 @@
       <Filename>eog.desktop</Filename>
       <Filename>evince.desktop</Filename>
       <Filename>file-roller.desktop</Filename>
-      <Filename>gcalctool.desktop</Filename>
       <Filename>gnome-dictionary.desktop</Filename>
       <Filename>gnome-disks.desktop</Filename>
       <Filename>gnome-font-viewer.desktop</Filename>
@@ -242,13 +241,13 @@
         <Not><Category>Settings</Category></Not>
         <Not><Category>X-GNOME-Utilities</Category></Not>
       </And>
+      <Filename>subscription-manager-gui.desktop</Filename>
+      <Filename>rhn_register.desktop</Filename>
     </Include>
 
     <Exclude>
       <!-- Utilities exclusions -->
       <Filename>baobab.desktop</Filename>
-      <Filename>gnome-system-log.desktop</Filename>
-      <Filename>gnome-system-monitor.desktop</Filename>
       <Filename>gnome-terminal.desktop</Filename>
 
       <!-- Sundry exclusions -->
@@ -282,19 +281,23 @@
       <Filename>eog.desktop</Filename>
       <Filename>evince.desktop</Filename>
       <Filename>file-roller.desktop</Filename>
-      <Filename>gcalctool.desktop</Filename>
       <Filename>gnome-dictionary.desktop</Filename>
       <Filename>gnome-disks.desktop</Filename>
       <Filename>gnome-font-viewer.desktop</Filename>
       <Filename>gnome-screenshot.desktop</Filename>
-      <Filename>gnome-system-log.desktop</Filename>
-      <Filename>gnome-system-monitor.desktop</Filename>
       <Filename>gnome-terminal.desktop</Filename>
       <Filename>gnome-tweak-tool.desktop</Filename>
       <Filename>gucharmap.desktop</Filename>
       <Filename>seahorse.desktop</Filename>
       <Filename>vinagre.desktop</Filename>
-      <Filename>yelp.desktop</Filename>
+    </Include>
+  </Menu>
+
+  <Menu>
+    <Name>Documentation</Name>
+    <Directory>Documentation.directory</Directory>
+    <Include>
+      <Category>Documentation</Category>
     </Include>
   </Menu>
 
